src/libraries/System.Text.Json/src/System/Text/Json/JsonConstants.cs

```diff
@@ -70,7 +70,6 @@ internal static partial class JsonConstants
 
         public const int MaxEscapedTokenSize = 1_000_000_000;   // Max size for already escaped value.
```

Comment:
Could we get rid of this limit too? FWIW I skimmed through the code and it looks like the recently added `WriteRawValue` methods completely disregard these limits.

Reply:
I'd like to get rid of all of the hardcoded limits.  But let's do them individually.